scripts/unicode.py

```diff
@@ -97,12 +141,244 @@ def load_confusables(f):
             raise Exception('More than one code point in first column')
         d_input = int(d_inputs[0].strip(), 16)
         for d_output in m.group(2).split():
-            d_outputitem = int(d_output, 16);
-            d_outputs.append(d_outputitem);
+            d_outputitem = int(d_output, 16)
+            d_outputs.append(d_outputitem)
         confusables.append((d_input, d_outputs))
 
     return confusables
 
+def aliases():
```

Comment:
perhaps link to the original source in unicode-script so things can be manually updated if necessary

Reply:
Added comments below this line.